pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1462,18 +1586,8 @@
     ^+  event-core
     %-  %^  trace  for.veb  sndr.packet
         |.("forward: {<sndr.packet>} -> {<rcvr.packet>}")
-    ::  set .origin.packet if it doesn't already have one, re-encode, and send
     ::
-    =?    origin.packet
-        &(?=(~ origin.packet) !=(%czar (clan:title sndr.packet)))
-      ?:  ?=(%& -.lane)
-        ~
-      ?.  (lte (met 3 p.lane) 6)
-        ~|  ames-lane-size+p.lane  !!
-      `p.lane
-    ::
-    =/  =blob  (encode-packet packet)
-    (send-blob & rcvr.packet blob)
+    (send-packet & `lane packet)
```

Comment:
I don't see a caller of `on-hear-forward` which provides `lane`- am I might be missing something?

Reply:
on-hear-forward gets called in on-hear-packet, with the same sample as on-hear-packet.
```
 %.  +<
 ::
 ?.  =(our rcvr.packet)
    on-hear-forward
```

Reply:
Got it, thanks

Reply:
Tracing this back: it seems that +call correctly passes lane when dud is a cell by extracting it from the task `lane.task` on line 1065. Otherwise is passes `lane` (line 1071) which I don't see in the sample at that point - is it passing the type `$lane` and not the value?

Reply:
It's still getting the lane and blob from the task: `[lane blob ~]:task`. The dud looks like a stack trace in case of a crash (I think in the arvo kernel).